ci/tasks/migrate.yml

```diff
@@ -0,0 +1,19 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: cybojenix/govwifi-concourse-runner
```

Comment:
Should this point to the Alphagov repo?

Reply:
This relates to a docker hub repository, so no.

The image_resource should probably be removed, as we pass it via the pipeline now